eZ/Publish/API/Repository/UserService.php

```diff
@@ -164,7 +164,8 @@ public function loadUserByCredentials($login, $password);
     /**
      * Loads a user for the given login.
      *
-     * Note: This method loads user by $login where $login might be case in-sensitive on certain storage engines!
+     * Since 6.1 $login is handled case insensitive across all storage engines and database backends, like was the case
```

Comment:
Ok, but regarding `case-insensitive` I changed this to `case insensitive` after feedback from Joao:
https://github.com/ezsystems/ezpublish-kernel/pull/1561#discussion_r50821173